meteor/lib/collections/TriggeredActions.ts

```diff
@@ -23,6 +23,8 @@ export interface DBTriggeredActions extends ProtectedStringProperties<IBlueprint
 	triggers: DBBlueprintTrigger[]
 
 	_rundownVersionHash: string
+
+	useLabelAsName?: boolean
```

Comment:
Is there a reason for this to be optional?

Reply:
Easier backwards compatibility? 
Since it's a change in Core I think NRK would appreciate that they don't have to go update a lot of stuff when we sync.
Whether that's a good reason for making something optional I'm not so sure...

Also, my migration defaults them to false anyway, so might as well not make this optional

Reply:
I wrote this comment before seeing the migration. But since there is a migration, as you mention yourself, the optionality is unnecessary.

Reply:
I'm not against having optional properties for rarely enabled features, because it seems like with every property like that, it's just a bit less data to store in the database. I don't know if (and by how much) that can really affect performance, but it might. In that case the migration is not necessary. But then shouldn't unchecking the checkbox unset the value instead of setting it to false?